fdbrpc/include/fdbrpc/fdbrpc.h

```diff
@@ -811,8 +811,13 @@ class RequestStream {
 			Future<Void> disc =
 			    makeDependent<T>(IFailureMonitor::failureMonitor()).onDisconnectOrFailure(getEndpoint());
 			auto& p = getReplyPromiseStream(value);
-			if (disc.isReady()) {
-				p.sendError(request_maybe_delivered());
+			// FIXME: buggify only in simulation/not during speed up simulation?
+			if (disc.isReady() || BUGGIFY_WITH_PROB(0.01)) {
```

Comment:
The unauthorized_attempt is an unrelated fix, this happened on failure/disconnect in tryGetReply so i assumed it should also be here. It looks like @sfc-gh-mpilman  added the original in tryGetReply